src/components/Plans/PlanCalculator/index.js

```diff
@@ -24,112 +24,119 @@ import { Slider } from './Slider';
+    const selectedPlanType = getPlanTypeFromUrlSegment(planTypeUrlSegment);
+    const sessionPlan = appSessionRef.current.userData.user;
 
-  useEffect(() => {
-    const fetchData = async () => {
-      try {
-        dispatch({ type: PLAN_TYPES_ACTIONS.FETCHING_STARTED });
-        const _planTypes = await planService.getPlanTypes();
-        dispatch({ type: PLAN_TYPES_ACTIONS.RECEIVE_PLAN_TYPES, payload: _planTypes });
-      } catch (error) {
-        dispatch({ type: PLAN_TYPES_ACTIONS.FETCH_FAILED });
-      }
-    };
-    fetchData();
-  }, [planService]);
+    useEffect(() => {
+      const fetchData = async () => {
+        try {
+          dispatch({ type: PLAN_TYPES_ACTIONS.FETCHING_STARTED });
+          const _planTypes = await planService.getPlanTypes();
+          dispatch({ type: PLAN_TYPES_ACTIONS.RECEIVE_PLAN_TYPES, payload: _planTypes });
+        } catch (error) {
+          dispatch({ type: PLAN_TYPES_ACTIONS.FETCH_FAILED });
+        }
+      };
+      fetchData();
+    }, [planService]);
+
+    useEffect(() => {
+      const fetchPlansByType = async () => {
+        try {
+          dispatchPlansByType({ type: PLANS_BY_TYPE_ACTIONS.FETCHING_STARTED });
+          const _plansByType = await planService.getPlansByType(
+            getPlanTypeFromUrlSegment(planTypeUrlSegment),
+          );
+          dispatchPlansByType({
+            type: PLANS_BY_TYPE_ACTIONS.RECEIVE_PLANS_BY_TYPE,
+            payload: _plansByType,
+          });
+        } catch (error) {
+          dispatchPlansByType({ type: PLANS_BY_TYPE_ACTIONS.FETCH_FAILED });
+        }
+      };
 
-  useEffect(() => {
-    const fetchPlansByType = async () => {
-      try {
-        dispatchPlansByType({ type: PLANS_BY_TYPE_ACTIONS.FETCHING_STARTED });
-        const _plansByType = await planService.getPlansByType(
-          getPlanTypeFromUrlSegment(planTypeUrlSegment),
-        );
-        dispatchPlansByType({
-          type: PLANS_BY_TYPE_ACTIONS.RECEIVE_PLANS_BY_TYPE,
-          payload: _plansByType,
-        });
-      } catch (error) {
-        dispatchPlansByType({ type: PLANS_BY_TYPE_ACTIONS.FETCH_FAILED });
+      if (planTypes.length > 0) {
+        setSelectedPlanIndex(INITIAL_VALUE_OF_SLIDER);
+        fetchPlansByType();
       }
-    };
+    }, [planService, planTypeUrlSegment, planTypes]);
 
-    if (planTypes.length > 0) {
-      setSelectedPlanIndex(INITIAL_VALUE_OF_SLIDER);
-      fetchPlansByType();
-    }
-  }, [planService, planTypeUrlSegment, planTypes]);
+    useEffect(() => {
+      setActiveClass('');
+      createTimeout(() => setActiveClass('active'), 0);
+    }, [planTypeUrlSegment, createTimeout]);
 
-  useEffect(() => {
-    setActiveClass('');
-    createTimeout(() => setActiveClass('active'), 0);
-  }, [planTypeUrlSegment, createTimeout]);
+    const handleSliderChange = (e) => {
+      const { value } = e.target;
+      const _selectedPlanIndex = parseInt(value);
+      setSelectedPlanIndex(_selectedPlanIndex);
+    };
```

Comment:
I don't understand why when running prettier these corrections were made

Reply:
![image](https://user-images.githubusercontent.com/1157864/140308462-592e718c-a37a-4a0f-a6ac-627058ee0351.png)

The new indentation has been created because line 27 has been splitter into two lines:

```
- export const PlanCalculator = InjectAppServices(({ dependencies: { planService } }) => {
+ export const PlanCalculator = InjectAppServices(
+  ({ dependencies: { planService, appSessionRef } }) => {
```

So, all the inner block has a new indentation level.

In my opinion, Prettier is wrong here, because the logical nesting level is the same. But, I understand that it is _arguable_ and Prettier is _opinionated_ 😛